.github/workflows/tag-merged-pr.yml

```diff
@@ -0,0 +1,15 @@
+on:
+  pull_request:
+    branches: 
+      - master
+    types: [closed]
+
+jobs:
+  my-action:
+    if: ${{ github.event.pull_request.merged }}
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
```

Comment:
I'd assume it shouldn't actually be necessary to checkout for this action.

Reply:
Indeed!